includes/Contracts/Feature.php

```diff
@@ -48,6 +48,15 @@ public function get_label(): string;
 	 */
 	public function get_description(): string;
 
+	/**
+	 * Gets the ability slug for the feature.
+	 *
+	 * @since 0.1.0
+	 *
+	 * @return string The ability slug for the feature.
+	 */
+	public function get_ability_slug(): string;
```

Comment:
Related to the above (partially depends on that decision): Do we really know that every single feature will have an ability? And only one? Could there never be a feature that has multiple abilities attached?

Reply:
> This could either be renamed (something like get_base_ability_slug) or maybe better to just move this to the ability class.

Third option is not abstracting it at all, and letting the Features decide how/where/when they are using the abilities API. 

Personally, I recommend this _regardless_ of the outcome of the other thread. We control the codebase, IMO there's no reason to abstract and add this level of prescribed coupling. If -after we have a handful of Features implemented - we see this is a pattern worth enforcing, we can do it then.

Reply:
Removed in https://github.com/WordPress/ai/pull/61/commits/779d3d6c711ef61f134617ae84bacaf651cc86f8